### PR TITLE
update to fix win-backup issue

### PR DESCRIPTION
**Pull request by mmadsen340**
Made some minor changes to add additional policies to the IAM role that runs Lambda, also updated the arn to reference the iam role profile vs the role for use with the win_lambda python script.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -69,6 +69,18 @@ resource "aws_iam_role_policy_attachment" "ec2ssm" {
   policy_arn = "arn:aws:iam::aws:policy/service-role/AmazonEC2RoleforSSM"
 }
 
+resource "aws_iam_role_policy_attachment" "ec2fullaccess" {
+  count      = "${var.include == "true" ? 1 : 0}"
+  role       = "${aws_iam_role.BsiBackup.name}"
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEC2FullAccess"
```
Comment (denniswebb) [inline main.tf]: I don't believe any and all EC2's should have full access to the EC2 API. It should have the minimum required to read the tags from itself and it's volumes, plus whatever it needs to create a snapshot.

File: main.tf [hunk 2]
```diff
@@ -69,6 +69,18 @@ resource "aws_iam_role_policy_attachment" "ec2ssm" {
...
+}
+
+resource "aws_iam_role_policy_attachment" "ssmfullaccess" {
+  count      = "${var.include == "true" ? 1 : 0}"
+  policy_arn = "${aws_iam_role.BsiBackup.name}"
+  role       = "arn:aws:iam::aws:policy/AmazonSSMFullAccess"
```
Comment (denniswebb) [inline main.tf]: Full access is too much here also. Doesn't the instance only need permissions to read from SSM?

File: output.tf [hunk 3]
```diff
@@ -5,4 +5,8 @@ output "backup_role_arn" {
 
 output "backup_role_name" {
   value = "${element(concat(aws_iam_role.BsiBackup.*.name, list("")), 0)}"
+}
+
+output "backup_role_profile_Arn" {
```
Comment (denniswebb) [inline output.tf]: Can we rename to `backup_instance_profile_arn`

File: main.tf (hunk 1 above)
Comment (denniswebb) [reply main.tf]: I see after further review there are 2 attachments above that should handle this. Why are the extra rights required?

**Comment by jcleve** [comment]
a little busy for a PR right now guys, but I'll try to find time later. PS. Where's my taco money?

**Pull request closed without merging**